examples/laplace_demo/laplace.ipynb

```diff
@@ -1,21 +1,21 @@
 {
```

Comment:
<p>This language needs work. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyDP/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Uhmm 😅 I haven't actually changed any of this. But sure I'll look into this as well.</p>

Reply:
> @chinmayshah99 I've updated the conclusion section in [da7990e](https://github.com/OpenMined/PyDP/commit/da7990ef4001d84f4473db9e581814a81b74ab5e). LMK if it looks okay.


Can we keep the existing part of the conclusion too as that is the technical summary?

Reply:
Uhm sure I'll add those back

Reply:
@chinmayshah99 how does it look after 64ba683